flink-kubernetes/src/main/java/org/apache/flink/kubernetes/KubernetesClusterClientFactory.java

```diff
@@ -66,6 +67,7 @@ public String getClusterId(Configuration configuration) {
 	}
 
 	private String generateClusterId() {
```

Comment:
The default schema of UUID is `e4225fc5-e4a1-4a81-b9b5-0b48baeea7a1`. Maybe it already meets our requirements. Also leaving the dash `-` could help to keep the name in a human readable pattern.

Reply:
Hi @wangyang0918!  The previous pattern is `flink-cluster-c7ce2409-1542-42d7-9843-ecd64277ea9e`, after this change, the pattern will be something like `flink-cluster-c7ce2409154242d79843ecd64277ea9`, which seems more human-readable. Personally I think it does not make too much sense to obey the default schema of UUID.
But if you insist on the previous pattern, I think we could add the `-` back.

Reply:
It is just my personal preference. You could keep the current schema if other reviews have no suggestion.